### ISSUE TITLE
Describe how to get the most recent R version

### ISSUE THREAD

**Issue by mikej888** [riboviz/riboviz#163]
From a user...

There is one thing that a newbie like me encountered in installing ribo-viz and was not soluble from the directions on github. That was to get the most updated R package. apt claimed the R 3.4.x version I had was the most recent version and didn't need updating. This might be worth noting in troubleshooting - alternative ways to get the most recent R version
Looking online, I saw that one way to get past this was to edit the sources list.
https://cran.r-project.org/bin/linux/ubuntu/README.html

However, while I could do this, since I didn't fully understand what other ramifications that might have, I did not. Instead, I found this way to get the most recent version of R:

Applying the protocol from this site: https://tecnstuff.net/how-to-install-r-on-ubuntu-18-04/ got me to R 3.6.3 and then all was right with the world, every subsequent component for ribo-viz installed fine

```console
$ sudo apt install dirmngr apt-transport-https ca-certificates software-properties-common gnupg2
$ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
$ sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/'
$ sudo apt update
$ sudo apt install r-base
```

**Comment by mikej888** [comment]
Replication on a fresh Ubuntu 18.04 VM:

See what R packages would be installed by `apt-get`:

```console
$ sudo apt-get -s install r-base
...
Conf r-base-core (3.4.4-1ubuntu1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-boot (1.3-20-1.1 Ubuntu:18.04/bionic [all])
Conf r-cran-cluster (2.0.6-2build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-foreign (0.8.69-1build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-mass (7.3-49-1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-kernsmooth (2.23-15-3build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-lattice (0.20-35-1build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-nlme (3.1.131-3build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-matrix (1.2-12-1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-mgcv (1.8-23-1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-survival (2.41-3-2build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-rpart (4.1-13-1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-class (7.3-14-2build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-nnet (7.3-12-2build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-spatial (7.3-11-2build1 Ubuntu:18.04/bionic [amd64])
Conf r-cran-codetools (0.2-15-1.1 Ubuntu:18.04/bionic [all])
Conf r-recommended (3.4.4-1ubuntu1 Ubuntu:18.04/bionic [all])
Conf r-base (3.4.4-1ubuntu1 Ubuntu:18.04/bionic [all])
Conf liblzma-dev (5.2.2-1.3 Ubuntu:18.04/bionic [amd64])
Conf r-base-dev (3.4.4-1ubuntu1 Ubuntu:18.04/bionic [all])
Conf r-doc-html (3.4.4-1ubuntu1 Ubuntu:18.04/bionic [all])
Conf r-base-html (3.4.4-1ubuntu1 Ubuntu:18.04/bionic [all])
```

Follow CRAN-R's [UBUNTU PACKAGES FOR R](https://cran.r-project.org/bin/linux/ubuntu/README.html).

Get my Ubuntu version:

```console
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04 LTS
Release:	18.04
Codename:	bionic
```

Add corresponding entry to `/etc/apt/sources.list`:

```
deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/
```

Update packages:

```console
$ sudo apt-get update
Hit:1 http://security.ubuntu.com/ubuntu bionic-security InRelease
Get:2 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ InRelease [3,626 B]
Hit:3 http://us.archive.ubuntu.com/ubuntu bionic InRelease 
Err:2 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 51716619E084DAB9
Hit:4 http://us.archive.ubuntu.com/ubuntu bionic-updates InRelease
Hit:5 http://us.archive.ubuntu.com/ubuntu bionic-backports InRelease
Reading package lists... Done                      
W: GPG error: https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 51716619E084DAB9
E: The repository 'https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Follow CRAN-R's [UBUNTU PACKAGES FOR R](https://cran.r-project.org/bin/linux/ubuntu/README.html) and install key:

```console
$ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
Executing: /tmp/apt-key-gpghome.8ibpbLvrIg/gpg.1.sh --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
gpg: key 51716619E084DAB9: public key "Michael Rutter <marutter@gmail.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

Update packages:

```console
$ sudo apt-get update
Hit:1 http://us.archive.ubuntu.com/ubuntu bionic InRelease
Hit:2 http://security.ubuntu.com/ubuntu bionic-security InRelease
Get:3 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ InRelease [3,626 B]
Hit:4 http://us.archive.ubuntu.com/ubuntu bionic-updates InRelease             
Hit:5 http://us.archive.ubuntu.com/ubuntu bionic-backports InRelease           
Get:6 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ Packages [77.1 kB]
Fetched 80.8 kB in 1s (114 kB/s)                         
Reading package lists... Done
```

See what R packages would now be installed by `apt-get`:

```console
$ sudo apt-get -s install r-base
...
Conf r-base-core (3.6.3-1bionic CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-boot (1.3-24-1cran1bionic0 CRAN:18.04/bionic-cran35 [all])
Conf r-cran-cluster (2.1.0-2bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-foreign (0.8.75-1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-mass (7.3-51.5-1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-kernsmooth (2.23-16-1+1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-lattice (0.20-40-1cran1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-nlme (3.1.144-1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-matrix (1.2-18-1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-mgcv (1.8-31-1cran1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-survival (3.1-11-1cran1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-rpart (4.1-15-2bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-class (7.3-16-1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-nnet (7.3-13-1bionic0 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-spatial (7.3-11-2cranArtful0~ubuntu18.04.1~ppa1 CRAN:18.04/bionic-cran35 [amd64])
Conf r-cran-codetools (0.2-16-1bionic0 CRAN:18.04/bionic-cran35 [all])
Conf r-recommended (3.6.3-1bionic CRAN:18.04/bionic-cran35 [all])
Conf r-base (3.6.3-1bionic CRAN:18.04/bionic-cran35 [all])
Conf liblzma-dev (5.2.2-1.3 Ubuntu:18.04/bionic [amd64])
Conf r-base-dev (3.6.3-1bionic CRAN:18.04/bionic-cran35 [all])
Conf r-base-html (3.6.3-1bionic CRAN:18.04/bionic-cran35 [all])
Conf r-doc-html (3.6.3-1bionic CRAN:18.04/bionic-cran35 [all])
222
```

Install:

```console
$ sudo apt-get -s install r-base
$ R --version
R version 3.6.3 (2020-02-29) -- "Holding the Windsock"
Copyright (C) 2020 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)
...
$ Rscript --version
R scripting front-end version 3.6.3 (2020-02-29)
```

The user's approach, of doing:

```console
$ sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/'
```

has the same effect as editing

**Comment by mikej888** [comment]
Added 'Troubleshooting: the most recent version of R is not installed' section to [docs/user/install.md](https://github.com/riboviz/riboviz/blob/get-current-r-163/docs/user/install.md) in  [get-current-r-163](https://github.com/riboviz/riboviz/tree/get-current-r-163) branch.
